src/System.Management.Automation/engine/Modules/ModuleUtils.cs

```diff
@@ -47,6 +47,7 @@ static ModuleUtils()
                     | (FileAttributes)FILE_ATTRIBUTE_RECALL_ON_DATA_ACCESS
                     | (FileAttributes)FILE_ATTRIBUTE_RECALL_ON_OPEN;
 
+                ApplicationInsightsTelemetry.SendUseTelemetry("PSModuleAutoLoadSkipOfflineFiles", "SkipOffLine");
```

Comment:
This can be removed now as the feature has been made stable.

Reply:
yes, we are collecting telemetry for stable features.